docs/usage.rst

```diff
@@ -9,13 +9,8 @@ Run locally
 Production setup
 ****************
 
-The easiest way would be to start the |tr| in 'prod' setup, as found in ``examples/development``.
-Run: ::
-
-  cd examples/production
-  ./run-prod.sh
-
-The production setup does not build images but downloads the images from docker hub.
+To start the |tr| without building any containers, start it as described in the `production example <https://github.com/knaw-huc/textrepo/tree/master/examples/production>`_.
```

Comment:
Moet de URL niet eindigen op een file? Nu kom je uit in de production example dir met alle files.
Bedoel je te linken naar 'start-prod.sh' ?

Reply:
Daar staat straks (na de merge) de README.md die github standaard laat zien:
https://github.com/knaw-huc/textrepo/tree/prod-example/examples/production

Reply:
👍